docs/core-token.md

```diff
@@ -22,6 +22,12 @@ token:
     retryInterval: 5s
     # retry to gain a lock on tokens this amount of times before failing the transaction
     numRetries: 3
+    # leaseExpiry is the period a token can be locked, after which it is forcefully unlocked
+    # if leaseExpiry is zero, the eviction algorithm is never executed
+    leaseExpiry: 3m
+    # leaseCleanupTickPeriod defines how often the eviction algorithm must be executed
+    # if leaseCleanupTickPeriod is zero, the eviction algorithm is never executed
+    leaseCleanupTickPeriod: 90s
```

Comment:
Shall we make the example the same as the default value?

Reply:
yes